pubspec.yaml

```diff
@@ -8,19 +8,19 @@ executables:
   markdown:
 
 environment:
-  sdk: '>=2.0.0-dev <2.0.0'
+  sdk: '>=2.0.0-dev <3.0.0'
 
 dependencies:
-  args: '^1.0.0'
-  charcode: '^1.1.0'
+  args: ^1.0.0
+  charcode: ^1.1.0
 
 dev_dependencies:
-  build_runner: ^0.8.2
+  build_runner: ^0.9.1
   build_web_compilers: ^0.4.0
-  collection: '^1.2.0'
-  expected_output: '^1.1.0'
+  collection: ^1.2.0
+  expected_output: ^1.1.0
   html: '>=0.12.2 <0.14.0'
-  js: '^0.6.1'
-  path: '^1.3.1'
+  js: ^0.6.1
+  path: ^1.3.1
   test: '>=0.12.42 <2.0.0'
```

Comment:
While you're here, make this `^1.2.0`

Reply:
We get this issue:
> Because every version of expected_output depends on test ^0.12.20 and markdown depends on test ^1.2.0, expected_output is forbidden.
    So, because markdown depends on expected_output ^1.1.0, version solving failed.

So I've submitted a PR for `expected_output`.

Reply:
https://github.com/dart-lang/expected_output/pull/19